src/Invoice_GenUI/Invoice_GenUI/ViewModels/CreateClientViewModel.cs

```diff
@@ -74,24 +47,32 @@ private void GoBack()
         [RelayCommand]
         private async Task CreateClient()
         {
-            var newClient = new CreateClientModel
-            {
-                ClientName = ClientName
-            };
-            var connected = await _clientService.PutClient(newClient);
-            bool result = connected;
-            if (result)
+            MessageBoxResult confirm = MessageBox.Show("Do you want to create this client?", "Confirm", MessageBoxButton.YesNo, MessageBoxImage.Question);
```

Comment:
:seedling: it will be worth looking at creating an injectable service which wraps around all of the calls to MessageBox.Show (but in a different branch and PR).

That way you can abstract the Windows-specific stuff away. And that will help with a potential migration to AvaloniaUI, as you would need to remove all Windows specific code to make it cross-platform.